### PR TITLE
Add Weekly Problem Set Description

### PR DESCRIPTION
**Pull request by Gansito144**
Add Weekly Problem Set Description
* Short Description about weekly problem set.
! TODO: Add right date for each contest.
! TODO: Provide more descriptive tags

### CODE REVIEW

File: app/components/ProblemSetBoard.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {Table, FormControl, FormGroup, ControlLabel } from 'react-bootstrap';
+import {ProblemSetData} from 'data/Data';
+
+export default class ProblemSetBoard extends React.Component {
+  constructor(props) {
```
Comment (rendon) [inline app/components/ProblemSetBoard.jsx]: Let's discuss indentation width offline and define our style guide.

File: app/components/ProblemSetBoard.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    super(props);
```
Comment (rendon) [inline app/components/ProblemSetBoard.jsx]: Let's also talk about `;`, to use or not to use, that's the question.

**Review by rendon — COMMENTED** [review]
Looks good, just a couple comments, let's discuss them offline and then we can merge.

**Comment by Gansito144** [comment]
@rendon I added the changes suggested. Please take a look at this PR.

**Comment by rendon** [comment]
Done! Keep them coming!

**Pull request merged**